src/plugins/loan-channel-query-map.js

```diff
@@ -170,37 +186,43 @@ export default {
 					id: 52,
 					url: 'loans-to-women',
 					queryParams: 'status=fundRaising&gender=female&riskRating=0,5',
-					algoliaParams: 'gender=female'
+					algoliaParams: 'gender=female',
+					flssLoanSearch: { gender: 'female' },
 				},
 				{
 					id: 53,
 					url: 'loans-for-education',
 					queryParams: 'status=fundRaising&riskRating=0,5&sector=15',
-					algoliaParams: 'sector=Education'
+					algoliaParams: 'sector=Education',
+					flssLoanSearch: { sector: [15] },
 				},
 				{
 					id: 54,
 					url: 'trending-now',
 					queryParams: 'status=fundRaising&riskRating=0,5',
-					algoliaParams: ''
+					algoliaParams: '',
+					flssLoanSearch: {},
 				},
 				{
 					id: 55,
 					url: 'loans-to-farmers',
 					queryParams: 'status=fundRaising&riskRating=0,5&sector=1',
-					algoliaParams: 'sector=Agriculture'
+					algoliaParams: 'sector=Agriculture',
+					flssLoanSearch: { sector: [1] },
 				},
 				{
 					id: 56,
 					url: 'loans-with-research-backed-impact',
 					queryParams: 'status=fundRaising&riskRating=0,5',
-					algoliaParams: ''
+					algoliaParams: '',
+					flssLoanSearch: {},
```

Comment:
This one is based on a different sort param for impact that isn't yet compatible with FLSS so it can't be included either.

Reply:
Looks like some of these are missing some params for Algolia. Removed.